### PR TITLE
Feature: Add to Calendar #5072

### PR DESCRIPTION
**Pull request by akash2408**
Fixes #

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
We're not using npm, please remove package-lock.json

File: app/templates/components/public/social-links.hbs [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <div class="ui list">
+  <div style="margin-bottom:4px;"><li>{{header-date @eventtime}} {{@eventtimezone}} To <span>{{header-date @eventtimeends}} {{@eventtimezone}}</span></li>
+  <a href="https://calendar.google.com/calendar/u/0/r/eventedit?dates={{gmt-time @eventtime}}/{{gmt-time @eventtimeends}}&text={{@eventname}}&location={{@eventlocation}}&sf=true">Add To Calender</a>
```
Comment (iamareebjamal) [inline app/templates/components/public/social-links.hbs:3]: No need of creating a new helper. Just create a computed property in the controller which returns this URL

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/helpers/gmt-time.js  2
- tests/integration/helpers/gmt-time-test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6397088&bid=20743030)

File: app/helpers/gmt-time.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Helper from '@ember/component/helper';
+import moment from 'moment';
+
+export function gmtTimeJs(params) {
+  const timezone = params[1] ? params[1] : moment.tz.guess();
+  return `${moment(params[0]).tz(timezone).utc().format('YYYYMMDD[T]HHmmSS[Z]')}`;
```
Comment (iamareebjamal) [inline app/helpers/gmt-time.js:6]:
```js
`${`${`${moment(abc)}`}`}` === moment(abc)
```

File: app/templates/components/public/social-links.hbs (hunk 1 above)
Comment (akash2408) [reply app/templates/components/public/social-links.hbs:3]: ohk

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/helpers/gmt-time.js  2
- tests/integration/helpers/gmt-time-test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6397088&bid=20743030)

**Pull request closed without merging**